ingestion/src/metadata/utils/engines.py

```diff
@@ -62,14 +73,14 @@ def create_generic_engine(connection, verbose: bool = False):
 
 
 @singledispatch
-def get_engine(connection, verbose: bool = False) -> Engine:
+def get_connection(connection, verbose: bool = False) -> Engine:
```

Comment:
hi sorry now I'm just being picky 😅  Could we maybe rename the module from `engines` to `connections`? Otherwise feels a bit misleading 🙏

Reply:
Also the signature, it does not always return an `Engine`. We can create a custom type that is the union of an `Engine`, `AWSClient` and any other connection object we might need